Client/src/components/contanier/staff/Courses/AddCourse/index.js

```diff
@@ -1,104 +1,186 @@
 import React, { Component } from "react";
+import { toast } from "react-toastify";
 import "./index.css";
 import LabeledInput from "../../../../common/LabeledInput";
 import Button from "../../../../common/Button";
+import { addCourseSchema, addDatesSchema } from '../../../../../helpers/validation-schema';
+
+const initState={
+  title: '',
+  description: '',
+  level: 0,
+  numberOfStudent: 0,
+  start: 0,
+  end: 0,
+  days: '',
+  Error: {},
+  dates: []
+}
 
 export default class Addcourse extends Component {
+  state = {
```

Comment:
@Fatmasiam You could change this to `state = initState` and reduce some of the duplication.  Then if you need to change the initial state, you only have to do it one place instead of two (.. and you won't forget to change the second place)